### ISSUE TITLE
ds is not defined

### ISSUE THREAD

**Issue by chriscurnow** [Wakanda/wakanda-issues#92]
I'm writing some code I want to test by running directly from Studio. I have the Debugger turned on.

The code calls a data class method. (Basically I just want to see that the method returns the correct result.)

I keep running into this problem where it comes to call the method (e.g.:

`ds.myClass.myMethod()`)

The debugger says `ds is not defined`.

The only way to get out of this seems to be by stopping then restarting the server.

**Comment by fredGandelot** [comment]
Hi Chris and sorry for the delay. 
To help me reproducing  faster could you tell me which Wakanda version are you currently using ?
Anyhow i test it and get back to you asap.
Fred

**Comment by fredGandelot** [comment]
Hi @chriscurnow ,

I've done few tests and I'm not able to reproduce your issue.

- I've tested a wak 1.x version on MAC
- I have a Dataclass1 with 2 methods  (one for the class, the other one is bound to an entity)
- The Debugger is On
- After creating the methods be sure that u've reloaded the model
- On a backend js file I just code ds.Dataclass1.method1() or ds.Dataclass1(1).methodEntity() and it works well.

Just to be sure, you're trying to execute your method on the backend, no ? (i.e not via REST nor on the Client Side)

**Comment by chriscurnow** [comment]
Thanks for following up on this issue.

I should have been more clear.

This does not happen all the time. But it does happen relatively regularly when I am testing some code and after and usually after I have had an error in a previous execution.

Yes I am running the code on the backend.

I have the method open in Studio and I select "Run"

**Comment by chriscurnow** [comment]
Here is an example.

The screenshot is from the debugger. This function is called from another function. This is the first time during execution that **ds** is referenced

I ran the calling function a few minutes ago but there was an error in one of the methods it called.

I fixed that error and ran it again. That's when I got the "**ds is not defined**" error.

The only way I have found out of this is is to stop and restart the server.

![ds-is-not-defined](https://cloud.githubusercontent.com/assets/2943047/19229689/2c7c0880-8f1b-11e6-840e-2a661929a4e9.png)
